src/ai/ai.cpp

```diff
@@ -101,6 +112,12 @@ void update_ai_general_status(sys::state& state) {
 				n.set_ai_rival(dcon::nation_id{});
 			}
 		}
+
+		if(n.get_ai_is_threatened() && !n.get_is_mobilized()) {
+			command::execute_toggle_mobilization(state, n);
```

Comment:
A nation mobilizing while at peace is a real bad idea. Even when at war, it will kill your economy and can make you lose the war if you can't buy enough supplies to recover org